setup.py

```diff
@@ -19,23 +19,16 @@ def requirements():
     """
     requirements_list = []
     for requirement in open("requirements.txt").read().rsplit("\n"):
-        if "python_version" in requirement:
-            if "> '3'" in requirement and int(sys.version_info[0]) == 3:
-                try:
+        if requirement != "":
```

Comment:
if not requirement should be enough i guess

Reply:
I think you misinterpreted this statement. If `requirement = "requests"` then as per statement it means it is not empty then follows the indented statement but if I change it to `if not requirement` that means requirement needs to empty  to follow the indented block.

But I can change it to `if requirement` if you want it.